modules/core/src/main/java/org/apache/ignite/internal/table/criteria/CursorAdapter.java

```diff
@@ -54,9 +52,9 @@ public CursorAdapter(AsyncCursor<T> ac) {
     @Override
     public void close() {
         try {
-            convertToPublicFuture(ac.closeAsync().toCompletableFuture()).join();
-        } catch (Throwable e) {
-            throw sneakyThrow(mapToPublicException(unwrapCause(e)));
+            ac.closeAsync().toCompletableFuture().join();
```

Comment:
AFAIK, we agree to use `future.get()` instead of `join`, because `join` is uninteruptable.

Reply:
Replaced with `future.get()`